### DISCUSSION TITLE
Add keybinding more zoom in & zoom out

### DISCUSSION THREAD

**Discussion by mahaveergurjar** [prasanthrangan/hyprdots#769] [Ideas]
In laptop Ctrl + touchpad two finger scroll up & down the window is zoom in & zoom out but not work on all app so make a keybind so this is also work in terminal and vscode

So think make this type keybind ctrl+scroll and connect this with another key ctrl+ and ctrl- zoom in zoom out
show this work in all app

**Comment by kRHYME7** [comment]
Have you seen some DE doing this ?

**Comment by mahaveergurjar** [comment]
I try and this is working 
See this link 
https://pointieststick.com/2023/03/08/did-you-know-you-can-metactrlscroll-to-zoom/#:~:text=Just%20hold%20down%20the%20Meta,didn't%20know%20about%20it.

**Reply by kRHYME7** [reply]
Sadly this  post is for Plasma Wayland

I think I will not go deeper. Maybe Hyprland does not support this including sway compositors( wlroots ) 

There is this one discussion here 
https://github.com/hyprwm/Hyprland/discussions/3467

As of now, Hyprland does not support zooming like a traditional desktop environment. While some Electron apps, like Visual Studio Code or Chromium-based browsers, do offer built-in zoom functionality, Hyprland does not manage these zoom levels in the same way.

This means that if you're using a tiling window manager like Hyprland, you won't be able to zoom in and out of individual windows or applications in the same way you can in a full-featured desktop environment.

I hope we can find more resources relating to this. Or you can issue a feature request on wlroots or Hyprland for now.

**Reply by kRHYME7** [reply]
by the way to zoom vscode and terminal you can use 

``` CTRL  = ``` to zoom in 
``` CTRL  - ``` zoom out



The ability to zoom depends on the application. Or the Operating system managing the Application in our case Hyprland

**Reply by mahaveergurjar** [reply]
Can you equally these key like 
Bind = ctrl+scrollup exec ctrl =
Bind= ctrl + scrolldown exect ctrl - 
Like this type so ctrl scroll work on all app
